andela/iceman-backend
# Set up pre-commit hooks

**What does this PR do?**

  This PR sets up pre-commit hooks

  **Description of Task to be completed?**

  I implemented pre-commit to enforce our coding standard and improve code quality

  **How should this be manually tested?**

 When you run `git push or git commit .. `, you will notice that linting rules will run and also your test. If any fails, you have to fix it before you can commit or push.

  **Any background context you want to provide?**

  N/A

  **What are the relevant stories?**

 N/A

  **Screenshots (if appropriate)**

<img width="533" alt="Screen Shot 2019-08-17 at 5 55 54 PM" src="https://user-images.githubusercontent.com/14803275/63215029-56bb4480-c118-11e9-8a19-fc17e80ec77e.png">


  **Questions:**

 N/A
